frontend/src/Page.js

```diff
@@ -69,9 +69,9 @@ function Page() {
     if (value.inclusion !== 'whole') {
       let fragment = '#' + value.inclusion;
       let imageReference = /!\[[^\]]*\]\([^)]+/;
-      return doc.text.replace(imageReference, '$&' + fragment);
+      doc.text.replace(imageReference, '$&' + fragment);
     }
-    return doc.text;
+    return doc.text.replace(')', ` "${doc.dc_title}")`);  
```

Comment:
Searching for closing parenthesis would work here, but doesn't seem to be very robust.
I think you should include your injection in the same `replace`.
However you will have to move the condition to change the value of the fragment.